clar2wasm/src/words/noop.rs

```diff
@@ -0,0 +1,130 @@
+use crate::wasm_generator::{GeneratorError, WasmGenerator};
+use clarity::vm::{ClarityName, SymbolicExpression};
+
+use super::Word;
+
+// Functions below are considered no-op's because they are instructions that does nothing
+// or has no effect when executed.
+// They only affect the types (in case of to-int and to-uint), and not the values.
+
+fn traverse_noop(
+    generator: &mut WasmGenerator,
+    builder: &mut walrus::InstrSeqBuilder,
+    args: &[SymbolicExpression],
+) -> Result<(), GeneratorError> {
+    generator.traverse_args(builder, args)?;
+
+    Ok(())
+}
+
+#[derive(Debug)]
+pub struct ToInt;
+
+impl Word for ToInt {
+    fn name(&self) -> ClarityName {
+        "to-int".into()
+    }
+
+    fn traverse(
+        &self,
+        generator: &mut WasmGenerator,
+        builder: &mut walrus::InstrSeqBuilder,
+        _expr: &SymbolicExpression,
+        args: &[SymbolicExpression],
+    ) -> Result<(), GeneratorError> {
+        traverse_noop(generator, builder, args)
```

Comment:
I know it's considered a "no-op" thingy, but we are still supposed to throw a runtime error if we have a number `> 2^127`, no ?

eg. `(to-int u340282366920938463463374607431768211455)` is a runtime error.

Reply:
Added an error guard for that case.

Reply:
Yep, you're right. I've simplified the implementation to pure Webassembly where the runtime error is straightforwad.